hw/ip/otbn/doc/_index.md

```diff
@@ -630,10 +630,93 @@ However, every error that OTBN detects when it isn't running is fatal.
 This means that the cause will be reflected in {{< regref "FATAL_ALERT_CAUSE" >}}, as described below in [Alerts]({{< relref "#alerts" >}}).
 This way, no alert is generated without setting an error code somewhere.
 
+### List of Errors {#design-details-list-of-errors}
+
+<table>
+  <thead>
+    <tr>
+      <th>Name</th>
+      <th>Type</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><code>BAD_DATA_ADDR<code></td>
+      <td>recoverable</td>
+      <td>A data memory access occurred with an out of bounds or unaligned access.</td>
+    </tr>
+    <tr>
+      <td><code>BAD_INSN_ADDR<code></td>
+      <td>recoverable</td>
+      <td>An instruction memory access occurred with an out of bounds or unaligned access.</td>
+    </tr>
+    <tr>
+      <td><code>CALL_STACK<code></td>
+      <td>recoverable</td>
+      <td>An instruction tried to pop from an empty call stack or push to a full call stack.</td>
+    </tr>
+    <tr>
+      <td><code>ILLEGAL_INSN<code></td>
+      <td>recoverable</td>
+      <td>
+        An illegal instruction was about to be executed.
+        An instruction is illegal in the following cases:
+        <ul>
+          <li>An instruction being excuted had an invalid encoding.</li>
+          <li>An access occurred for an invalid CSR or WSR.</li>
+        </ul>
```

Comment:
I'm wondering if we should remove this detailed list and refer to the ISA documentation instead, which now lists the error conditions in much more detail. (Here and below in the LOOP case.)

Reply:
Sounds like a good idea to me.